### ISSUE TITLE
Downsample too fast 

### ISSUE THREAD

**Issue by OrNot** [Paperspace/DinoRunTutorial#4]
The 3 layers of conv  with proposed values of stride , pooling .ect.  leads the feature map to be small too fast ( supposed 80*80 input). is there any other settings I missed ?

Thanks a lot

**Comment by ravi72munde** [comment]
Most of the data pixels in the image are empty so small feature map should not hinder the accuracy or speed. I personally believe stripping away some data was necessary as we are processing images real-time as we perform actions. I have in fact got satisfactory results with an image input of mere 20x40 as well. That being said, please try out and let us know if you find a better configuration. The current training configuration requires at least 1 million steps, which is quite OK in case of reinforcement learning but if we can reduce that somehow that's certainly an achievement.

**Comment by OrNot** [comment]
Thanks for your reply. I am following your codes and try to re-implement with pytorch. I modify  the model to align the architecture as proposed  in the paper  double-dqn.  I have run the loops nearly 20000 but the highest score  is still under 100.  Is it a normal training or should I cease the process  to double check my code  ?

**Comment by ravi72munde** [comment]
20000 is a small count to judge. I would suggest keep it for another 20000. If you see no progress, check your code.
